packages/nx/src/hasher/hasher.ts

```diff
@@ -222,7 +226,7 @@ class TaskHasher {
         return this.hashExternalDependency(task.target.project);
       }
       const namedInputs = {
```

Comment:
Maybe instead of exporting `DEFAULT_NAMED_INPUTS`, we can export the `getNamedInputs` from here:

```typescript
export function getNamedInputs(nxJson: NxJsonConfiguration, project: ProjectGraphProjectNode) {
	return {
        default: [{ fileset: '{projectRoot}/**/*' }],
        ...nxJson.namedInputs,
        ...project.data.namedInputs,
	};
}
```

That would keep the consistency between hasher and the project graph generation